plugins/gatsby-source-snooty-prod/gatsby-node.js

```diff
@@ -77,6 +57,38 @@ const createRemoteMetadataNode = async ({ createNode, createNodeId, createConten
   }
 };
 
+/**
+ * Creates graphql nodes on metadata for associated products
+ * association can be an umbrella product, associated product (in snooty.toml),
```

Comment:
(Super nit) Punctuation might be good here since JSDoc currently sees this as one long sentence.

```suggestion
 * Creates graphql nodes on metadata for associated products.
 * Association can be an umbrella product, associated product (in snooty.toml),
```

Reply:
done!